dali/image/tiff_libtiff.cc

```diff
@@ -260,6 +260,14 @@ TiffImage_Libtiff::TiffImage_Libtiff(const uint8_t *encoded_buffer,
     TIFFGetFieldDefaulted(tif_.get(), TIFFTAG_ROWSPERSTRIP, &rows_per_strip_));
   LIBTIFF_CALL(
     TIFFGetFieldDefaulted(tif_.get(), TIFFTAG_COMPRESSION, &compression_));
+
+  uint16_t photometric_interpretatation;
```

Comment:
```suggestion
  uint16_t photometric_interpretation;
```

Reply:
I'd also add a comment that for now we do not support palette images in this implementation. It can be here or in CanDecode

Reply:
Done